code/Model/EditableFormField/EditableCountryDropdownField.php

```diff
@@ -19,16 +24,40 @@ class EditableCountryDropdownField extends EditableFormField
 
     private static $plural_name = 'Country Dropdowns';
 
+    private static $db = array(
+        'UseEmptyString' => 'Boolean',
+        'EmptyString' => 'Varchar(255)',
+    );
+
     private static $table_name = 'EditableCountryDropdownField';
 
     /**
-     * @return FieldList
+     * @return \SilverStripe\Forms\FieldList
      */
     public function getCMSFields()
     {
         $fields = parent::getCMSFields();
 
         $fields->removeByName('Default');
+        $fields->addFieldToTab(
+            'Root.Main',
+            DropdownField::create('Default', _t(__CLASS__ . '.DEFAULT', 'Default value'))
+                         ->setSource(i18n::getData()->getCountries())
+                         ->setHasEmptyDefault(true)
+                         ->setEmptyString('---')
+        );
+
+        $fields->addFieldToTab(
+            'Root.Main',
+            CheckboxField::create('UseEmptyString')
+                         ->setTitle('Set default empty string')
```

Comment:
Looking good now! One last change request though; could you please make the Title for this field translatable? You can see an example of this in the `DropdownField` creation on line 44.